[0.18.0] missing file in pecl archive

```
/dev/shm/BUILD/php-pecl-datadog-trace-0.18.0/NTS/src/ext/serializer.c:7:10: fatal error: mpack/mpack.h: No such file or directory
 #include "mpack/mpack.h"
          ^~~~~~~~~~~~~~~
compilation terminated.

```

--- a/package.xml
+++ b/package.xml
@@ -82,6 +82,9 @@
                     <file name="closure_accessing_outside_variables.phpt" role="test" />
                     <file name="closure_set_inside_object_methods.phpt" role="test" />
                     <file name="dd_trace_serialize_msgpack.phpt" role="test" />
+                    <file name="dd_trace_serialize_msgpack_error.phpt" role="test" />
+                    <file name="dd_trace_serialize_msgpack_error_strict.phpt" role="test" />
+                    <file name="disable_tracing_disables_tracing.phpt" role="test" />
                     <file name="do_not_check_if_class_or_function_exists_by_default.phpt" role="test" />
                     <file name="enable_throw_exception_if_overridable_doesnt_exist.phpt" role="test" />
                     <file name="method_invoked_via_reflection.phpt" role="test" />
@@ -94,17 +97,24 @@
                     <file name="private_self_access.phpt" role="test" />
                     <file name="protected_method_hook.phpt" role="test" />
                     <file name="recursion.phpt" role="test" />
+                    <file name="request_init_hook_check_blacklisted_modules.phpt" role="test" />
                     <file name="request_init_hook_file_not_found.phpt" role="test" />
                     <file name="reset_configured_overrides.phpt" role="test" />
                     <file name="reset_function_tracing.phpt" role="test" />
                     <file name="return_value_passed.phpt" role="test" />
                     <file name="run_file_before_request_handling.phpt" role="test" />
+                    <file name="segfault_backtrace_disabled.phpt" role="test" />
+                    <file name="segfault_backtrace_disabled_via_env_enabled_in_ini.phpt" role="test" />
+                    <file name="segfault_backtrace_enabled.phpt" role="test" />
+                    <file name="segfault_backtrace_enabled_via_env_disabled_in_ini.phpt" role="test" />
+                    <file name="segfault_backtrace_enabled_via_env.phpt" role="test" />
                     <file name="simple_function_hook.phpt" role="test" />
                     <file name="simple_method_hook.phpt" role="test" />
                     <file name="simple_sanity_check.phpt" role="test" />
                     <file name="throw_exception.phpt" role="test" />
                     <file name="trace_method_or_function_that_will_exist_later.phpt" role="test" />
                     <file name="trace_static_method.phpt" role="test" />
+                    <file name="used_dispatch_shouldn_t_be_freed.phpt" role="test" />
                     <file name="variable_length_parameter_list.phpt" role="test" />
                     <file name="with_params_function_hook.phpt" role="test" />
                     <file name="with_params_method_hook.phpt" role="test" />

--- a/package.xml
+++ b/package.xml
@@ -63,6 +63,14 @@
                     <file name="serializer.c" role="src" />
                     <file name="serializer.h" role="src" />
                     <file name="version.h" role="src" />
+                    <dir name="mpack">
+                        <file name="AUTHORS.md" role="doc" />
+                        <file name="CHANGELOG.md" role="doc" />
+                        <file name="README.md" role="doc" />
+                        <file name="LICENSE" role="doc" />
+                        <file name="mpack.c" role="src" />
+                        <file name="mpack.h" role="src" />
+                    </dir>
                 </dir>
             </dir>
             <dir name="tests">

